freezegun/api.py

```diff
@@ -405,13 +410,15 @@ def today(cls):
 
     @classmethod
     def utcnow(cls):
-        result = cls._time_to_freeze() or real_datetime.utcnow()
+        if _should_use_real_time():
+            real_datetime.utcnow()
+
+        result = cls._time_to_freeze()
         return datetime_to_fakedatetime(result)
 
     @staticmethod
     def _time_to_freeze():
-        if freeze_factories:
```

Comment:
📝  length of `freeze_factories` is equal to that of `ignore_lists`, and this is [checked](https://github.com/spulec/freezegun/pull/374/files#diff-83feda307c275990ac39bb294a0c02cc99112d380c4ae3a5a9c326bd6533a2a0R171) in `_should_use_real_time`